lib/services/eventgridManagement/package.json

```diff
@@ -2,9 +2,9 @@
   "name": "azure-arm-eventgrid",
   "author": "Microsoft Corporation",
   "description": "EventGridManagementClient Library with typescript type definitions for node",
-  "version": "1.2.0",
+  "version": "1.1.0",
```

Comment:
The version number can't go backwards. Remember to increase the version number in the readme.nodejs.md file. Double-check on my comment above to determine whether this version should be `2.0.0` or `1.3.0`.